src/workforce-management/Views/Employee/Detail.cshtml

```diff
@@ -0,0 +1 @@
+﻿<h4>HELLO!</h4>

```

Comment:
I was using it to test the employee detail link, this will be useful once Dayne's branch is merged.